readthedocs/doc_builder/director.py

```diff
@@ -197,17 +206,31 @@ def build(self):
 
     # VCS checkout
     def checkout(self):
-        log.info(
-            "Clonning repository.",
-        )
+        """Checkout Git repo and load build config file."""
```

Comment:
💯 for adding new docstrings.

Reply:
RIP "clonning", I will miss "clonning" 😢

Reply:
```
$ ag clonning readthedocs
readthedocs/vcs_support/backends/git.py
208:            # after clonning it.
```

Still one left :smile: